### ISSUE TITLE
wakeword_backend "pvporcupine" is not free?

### ISSUE THREAD

**Issue by sangheonEN** [KoljaB/RealtimeSTT#147]
wakeword_backend: str = "pvporcupine",

If you look at the contents of https://pypi.org/project/pvporcupine/,

Create an instance of the engine

import pvporcupine

access_key = "${ACCESS_KEY}" # AccessKey obtained from Picovoice Console (https://console.picovoice.ai/)

handle = pvporcupine.create(access_key=access_key, keywords=['picovoice'])

When creating a model, access_key is required.

However, in the wakeword_test.py code of RealTimeSTT, access_key is not assigned when creating as shown below.

Are you providing a free version of the model yourself?

from RealtimeSTT import AudioToTextRecorder
import logging

if __name__ == '__main__':

    def recording_started():
        print("Speak now...")

    def recording_finished():
        print("Speech end detected... transcribing...")

    with AudioToTextRecorder(spinner=False, level=logging.DEBUG, model="small.en", language="en", wake_words="jarvis", on_wakeword_detected=recording_started, on_recording_stop=recording_finished
        ) as recorder:
        print('Say "Jarvis" then speak.')
        print(recorder.text())
        print("Done. Now we should exit.")

**Comment by KoljaB** [comment]
I am using version 1.9.5 of pvporcupine. This was the last free version before they went commercial.

**Comment by sangheonEN** [comment]
1.9.5 of pvporcupine, can I use the keyword I want to wake up with for free?

I don't have a model created, so I don't know how to create a model.

**Comment by KoljaB** [comment]
No, you can only select from a predefined list of keywords with this version.

**Comment by KoljaB** [comment]
For custom wake words please use the openwakeword engine

**Comment by sangheonEN** [comment]
Can you give me some tips on training an open wake word model?

I want to create a wake word model that is suitable for Korean speech, but training an open wake word model

config["target_phrase"] = ["hey sebastian"]

I think it's generated based on text, like target_phrase.

It's interesting that it's created with only text, without any speech data.

I wonder what kind of learning method would be best for generating a model. Can you give me some tips?

**Comment by KoljaB** [comment]
I’m no pro at training OpenWakeWord ONNX models. I just used the basic [Colab Notebook](https://colab.research.google.com/drive/1q1oe2zOyZp7UsB3jJiQ1IFn8z5YfjwEb?usp=sharing) from [this repo](https://github.com/dscripka/openWakeWord?tab=readme-ov-file#training-new-models) to train mine. It doesn’t actually generate anything straight from text. Instead, it uses Piper TTS to convert the input text into audio and trains the model on that. Can’t really give expert tips here since I’m not an expert myself. Replacing the piper generation with a better source probably would be a start.
